front-end/src/IndividualShow.css

```diff
@@ -0,0 +1,103 @@
+
```

Comment:
The CSS for individual show looks good, though I think there is room for improvement.

- There is a white border around everything that I think we should probably remove. 
- Add to in progress shows, add to watched shows, and return to shows should probably be buttons, not links.
- In terms of buttons, if you check the prof-button styling in Profile.css, I suggest you use the buttons provided there for all buttons throughout the project. They also have a hover style that would be good to implement.
- In general, I'd suggest only using colors from the color palette, which is in palette.png.
- For now, all redirects should probably send you to my-shows/1, as otherwise, they just send you to home.
- In terms of platforms, there's a list of platforms for the project available in the master branch's MyShows.js.
- I think Sam said this earlier, but the description of the show should probably not be an input box.
Good work so far.

Reply:
I didn't really have anywhere I could put this comment on GitHub, but poster.jpg should be in the Images folder.

Reply:
thanks, yes, in the newest commit, I updated the styling/buttons - let me know if that works.  I added platforms from MockData.js--is it ok or anything else is needed? I will try to remove white borders - its from linter (prettier); i'll update link to 'my-shows/1' page